### PR TITLE
Help developer install build-in necessary definitions.

### PR DESCRIPTION
**Pull request by wangyuan249**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Check whether the user install definitions before make core-run

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by wangyuan249** [comment]
cc @yangsoon

File: contribute/developer-guide.md [hunk 1]
```diff
@@ -78,8 +78,13 @@ Firstly make sure your cluster has CRDs, below is the command that can help inst
 make core-install
 ```
 
-Run locally:
+To ensure you have created vela-system namespace and install definitions of necessary module.
+you can run the shell script:
+```shell script
+sh hack/utils/installdefinition.sh
```
Comment (wonderflow) [inline contribute/developer-guide.md:84]: It should be a make command

**Comment by chivalryq** [comment]
Why not reccommand to use helm tool for better template rendering instead of doing it manually?

File: contribute/developer-guide.md (hunk 1 above)
Comment (wangyuan249) [reply contribute/developer-guide.md]: got it

File: hack/utils/installdefinition.sh [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+if [ -n "$res" ];then
+  echo 'vela-system namespace exist'
+else
+  echo 'vela-system namespace do not exist'
+  echo 'creating vela-system namespace ... '
+  kubectl create namespace vela-system
+fi
+echo "applying definitions ..."
+cd $DEFPATH
+# shellcheck disable=SC2045
```
Comment (wonderflow) [inline hack/utils/installdefinition.sh:14]: don't disable it

File: hack/utils/installdefinition.sh [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+for file in `ls .`
+  do
+    echo "Info: changing "$DEFPATH"/"$file
+    sed -i '' "s#{{.Values.systemDefinitionNamespace}}#vela-system#g" $file
+    kubectl apply -f $file
+    sed -i '' "s#vela-system#{{.Values.systemDefinitionNamespace}}#g" $file
```
Comment (wonderflow) [inline hack/utils/installdefinition.sh:20]: what if the file contains many `vela-system`?

Comment (wonderflow) [reply hack/utils/installdefinition.sh:20]: I suggest use `sed -i.bak`

1. different system has same command. both linux and macos
2. you just need to delete the backup file after apply

File: hack/utils/installdefinition.sh (hunk 2 above)
Comment (wangyuan249) [reply hack/utils/installdefinition.sh]: fixed

File: hack/utils/installdefinition.sh (hunk 3 above)
Comment (wangyuan249) [reply hack/utils/installdefinition.sh]: fix

**Review by wonderflow — APPROVED** [review]

**Pull request merged**